### ISSUE TITLE
Logo ideas

### ISSUE THREAD

**Issue by orta** [tsnyc/tsnyc.github.io#3]
I threw some time at logo idea. I feel like there's more to explore on the idea of `TS<NYC>` but it's a little hard on a square canvas. Will need to see what other JS meet ups do to see if there's any common patterns.

<img width="1861" alt="screen shot 2017-10-25 at 9 55 45 am" src="https://user-images.githubusercontent.com/49038/32002608-e86922e2-b96a-11e7-9edd-09a89d2ba18f.png">

https://cl.ly/3k1v333U0S3L

**Comment by danvk** [comment]
Is the idea that `TS<NYC>` is a generic type? I've heard (maybe through @kirjs) that the empire state building is extremely litigious about use of their building outline.

**Comment by orta** [comment]
Yeah, that TS is generic and we're scoping it to NYC. 
I wonder if something like  `I :heart: TS` could work as well

**Comment by orta** [comment]
Think I'm pretty sold on:

<img width="284" alt="screen shot 2017-11-05 at 12 18 09 pm" src="https://user-images.githubusercontent.com/49038/32417185-92b63d46-c223-11e7-9b1b-e5bb297df55d.png">

**Comment by erindepew** [comment]
Personally, I like the TypeScript<NYC> a lot, I think it's very clever. I riffed off your designs a bit, I think it makes sense to use the original TS logo color (#1675bc) and font (Segoe UI) so it gives a nod to TypeScript.

![screen shot 2017-11-05 at 9 14 12 pm](https://user-images.githubusercontent.com/3421197/32422455-5e1fd646-c26e-11e7-8c67-e68a63203235.png)

[logos-v2.ai.zip](https://github.com/tsnyc/tsnyc.github.io/files/1444731/logos-v2.ai.zip)

**Comment by orta** [comment]
Yeah, this looks great - the kerning feels a bit weird on `<NYC>` are there spaces there, or is the font just weird with it?

**Comment by erindepew** [comment]
@orta Yeah, I increased the tracking slightly because it felt a bit smushed. I can fiddle with it a bit more to make it sit a bit more evenly. Also just realized that most avatar/icons are now round so a square shaped one might not be the best idea, or we need a square that will also look fine as a circle. 

I'll take another stab at it tonight/tomorrow.

**Comment by erindepew** [comment]
@orta moved in the carets a bit and increased the kerning slightly on the actual text, so it should feel a bit more balanced now. 
Also wasn't sure which direction to go for the smaller, avatar logo, thoughts? 

<img width="738" alt="screen shot 2017-11-09 at 3 04 16 pm" src="https://user-images.githubusercontent.com/3421197/32627122-6f996394-c55f-11e7-9b3a-edeb565ece0d.png">

[logos-v3.ai.zip](https://github.com/tsnyc/tsnyc.github.io/files/1459035/logos-v3.ai.zip)

**Comment by orta** [comment]
Looks 👍  to me. 

I'm a big fan of

![screen shot 2017-11-09 at 5 12 50 pm](https://user-images.githubusercontent.com/49038/32632332-3cd8aa3e-c571-11e7-95d6-bd7fa53d6c04.png)

Feels solid and well-balanced.

**Comment by danvk** [comment]
Love it!

**Comment by erindepew** [comment]
😊  thanks guys, @danvk here's the logos as svgs and pngs and in black, white and color. That should cover any situation you might need these for. 
[TypescriptLogos.zip](https://github.com/tsnyc/tsnyc.github.io/files/1486111/TypescriptLogos.zip)
